content/numpy/concepts/ndarray/terms/transpose/transpose.md

```diff
@@ -52,3 +52,17 @@ This produces the following output:
  [2 5]
  [3 6]]
 ```
+
+In the above output, the original array is a 2x3 matrix, and the transposed array is a 3x2 matrix.
+
+## Codebyte example
+
+This codebyte example demonstrates how to create and manipulate a multi-dimensional array in Python using NumPy’s `transpose` function.
+
+```codebyte/python
+import numpy as np
+
+a = np.ones((2, 3, 4, 5))
+np.transpose(a).shape
+(5, 4, 3, 2)
+```
```

Comment:
The output in the last line is not generated by the code itself, so we will need to print the output to verify it directly:

```suggestion
```codebyte/python
import numpy as np

a = np.ones((2, 3, 4, 5))
print(np.transpose(a).shape)
```
```